src/components/event/ParticipantsSection.tsx

```
@@ -0,0 +1,60 @@
+              ]}
+              value={viewMode}
+              onChange={(value) => setViewMode(value)}
+            />
+          </div>
+        </div>
+
+        {viewMode === 'grid' && (
+          <ul className="mt-4 grid grid-cols-2 gap-y-12 md:grid-cols-3">
+            {Array.from({ length: 32 }).map((item) => (
+              <li className="flex flex-col items-center">
+                <div className="flex aspect-square h-32 flex-col items-center justify-center rounded-full border border-white/10 bg-white/10">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)